### PR TITLE
stuff

### PR DESCRIPTION
**Pull request by TomasTomecek**
let me know what you think

### CODE REVIEW

**Comment by bkabrda** [comment]
There's a minor issue with this, related to [1], let me explain into more detail.

tl;dr version: You need to fix the Dockerhub hook to `mv Dockerfile Dockerfile.template` and I'll need to adjust the automatic build setup at Dockerhub. Long version follows.

Dockerhub expects you to provide name of the Dockerfile to build from and expects this file to be in the repo *before any hooks are run*. Therefore we tell Dockerhub that it should build `Dockerfile` (when creating the automatic build there) even though it's just a template. Then, in [1], we make sure that *after* the rendering is done, we replace the template with the rendered `Dockerfile`. Hence, with this PR, we'd need to tell Dockerhub that it should build `Dockerfile.template` and alter the hook to `mv Dockerfile Dockerfile.template` for this to work.

I hope this make sense. Either way, this behaviour needs to be additionally documented in the README, which I'm happy to do after this PR is merged.

[1] https://github.com/bkabrda/container-image-template-dg/blob/master/hooks/pre_build#L4-L9

**Comment by bkabrda** [comment]
LGTM now, merging. Thanks!
